### PR TITLE
Update Table of contents and Command summary for UG

### PR DESCRIPTION
**Pull request by jinnieshin**
Adds Table of Contents with hyperlinks
Updates Command summary to match the current features Adds FAQ

### CODE REVIEW

**Review by lukkesreysandeur — CHANGES_REQUESTED** [review]

File: docs/UserGuide.md [hunk 1]
```diff
@@ -321,26 +346,38 @@ Exits the program.
...
-| **Deleting patient from system** | `delpat id/ID_NUMBER`<br> e.g., `delpat id/A0123456789B`                                                                                                                                                                   |
-| **Exit**                         | `exit`<br>                                                                                                                                                                                                                 |
-| **Clear**                        | `clear`<br>                                                                                                                                                                                                                |
+| Action                                                                                                        | Format, Examples                                                                         |
```
[the remark this answers is not in the document]
Comment (jinnieshin) [reply docs/UserGuide.md:357]: Wouldn't that be too lengthy for a summary..?

File: docs/UserGuide.md [hunk 2]
```diff
@@ -21,8 +21,33 @@ ___
 
 ## Table of Contents
 
-* Table of Contents
-{:toc}
```
[the remark this answers is not in the document]
Comment (jinnieshin) [reply docs/UserGuide.md:25]: Ok

File: docs/UserGuide.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (lukkesreysandeur) [reply docs/UserGuide.md:330]: Maybe we can split it into 3 columns instead? One for the example and one for the format.

File: docs/UserGuide.md [hunk 3]
```diff
@@ -20,7 +20,7 @@ reliably than traditional GUI based apps.
 ___
 
 ## Table of Contents
-- Table of Contents {:toc}
+* Table of Contents {:toc}
```
Comment (lukkesreysandeur) [inline docs/UserGuide.md:23]: Actually this line still doesn't generate the table correctly, the {:toc} should be on the next line.

**Review by euph00 — APPROVED** [review]
I will make the requested changes on my end

**Pull request merged**